src/compiler/c.rs

```diff
@@ -276,6 +291,7 @@ pub fn hash_key(compiler_digest: &str,
     let mut m = Digest::new();
     m.update(compiler_digest.as_bytes());
     m.update(CACHE_VERSION);
+    m.update(extension.as_bytes());
```

Comment:
I guess you should rev `CACHE_VERSION` here, since you're changing the hash inputs.